readme.md

```diff
@@ -53,7 +53,12 @@ body{
 }
 ```
 
-> Your answer...
+ The color red should not be called as a string, there is no need for quotes in CSS. And instead of a equal sign there should be a : colon in its place.
+Example
+body{
+  background-color: red;
+}
+
```

Comment:
:+1: 

The one time you would need quotes in CSS is if you're specifying a font name that has spaces in it, like `"Times New Roman"`.